src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,85 +1,124 @@
 package nextstep.subway.line;
 
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import nextstep.subway.AcceptanceTest;
+import nextstep.subway.line.dto.LineRequest;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import static nextstep.subway.line.LineScenarioMethod.*;
+
 @DisplayName("지하철 노선 관련 기능")
-public class LineAcceptanceTest extends AcceptanceTest {
+class LineAcceptanceTest extends AcceptanceTest {
+
+    private LineRequest 신분당선 = new LineRequest("신분당선", "bg-red-600");
+    private LineRequest 구분당선 = new LineRequest("구분당선", "bg-blue-600");
+    private LineRequest 수인선 = new LineRequest("수인선", "bg-yellow-600");
+
+
     @DisplayName("지하철 노선을 생성한다.")
     @Test
     void createLine() {
         // when
-        // 지하철_노선_생성_요청
+        ExtractableResponse<Response> response = 지하철_노선_생성_요청(신분당선);
 
         // then
-        // 지하철_노선_생성됨
+        지하철_노선_생성됨(response);
     }
 
-    @DisplayName("기존에 존재하는 지하철 노선 이름으로 지하철 노선을 생성한다.")
+    @DisplayName("지하철 노선이 이미 등록되어 있는 경우 지하철 노선 생성에 실패한다.")
     @Test
-    void createLine2() {
+    void createLineWithDuplicateName() {
         // given
-        // 지하철_노선_등록되어_있음
+        지하철_노선_등록되어_있음(신분당선);
 
         // when
-        // 지하철_노선_생성_요청
+        ExtractableResponse<Response> response = 지하철_노선_생성_요청(신분당선);
 
         // then
-        // 지하철_노선_생성_실패됨
+        지하철_노선_생성_실패됨(response);
     }
 
     @DisplayName("지하철 노선 목록을 조회한다.")
     @Test
-    void getLines() {
+    void findLines() {
         // given
-        // 지하철_노선_등록되어_있음
-        // 지하철_노선_등록되어_있음
+        지하철_노선_등록되어_있음(신분당선);
+        지하철_노선_등록되어_있음(수인선);
 
         // when
-        // 지하철_노선_목록_조회_요청
+        ExtractableResponse<Response> response = 지하철_노선_조회_요청("/lines");
 
         // then
-        // 지하철_노선_목록_응답됨
-        // 지하철_노선_목록_포함됨
+        지하철_노선_조회_응답됨(response);
+        지하철_노선_목록_조회_결과_포함됨(response, 신분당선);
+        지하철_노선_목록_조회_결과_포함됨(response, 수인선);
     }
 
     @DisplayName("지하철 노선을 조회한다.")
     @Test
-    void getLine() {
+    void findLine() {
+        // given
+        지하철_노선_등록되어_있음(수인선);
+
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_조회_요청("/lines/1");
+
```

Comment:
리팩토링 잘해주셨네요 👍
아래처럼 `리턴값`을 활용하여 넘기는게 좋겠네요.
다른 곳도 확인 부탁드립니다.

```suggestion
    void findLine() {
        // given
        String createdLocationUri = 지하철_노선_등록되어_있음(수인선);

        // when
        ExtractableResponse<Response> response = 지하철_노선_조회_요청(createdLocationUri);
        
```

Reply:
네~! 조회 부분이랑 수정, 제거 부분에 LocationUri 값을 활용해서 적용하였습니다.